### PR TITLE
Import code for simpleprophet models and tools.

### PR DESCRIPTION
**Pull request by jmccrosky**
This is an import from my code in dscontrib.  I'm not sure if I need to set up anything else so this is usable?  Comments appreciated.  Feel free to review the code itself if you like - I'm willing to do some cleaning and refactoring if needed.  No obligation though.

### CODE REVIEW

File: simpleprophet/data.py
Comment (jklukas) [inline simpleprophet/data.py]: I'd expect constants like these to be all caps like KPI_QUERIES, but also respect a leading underscore for "private" implementation details.

File: simpleprophet/__init__.py
Comment (jklukas) [inline simpleprophet/__init__.py:1]: We could use some basic documentation, perhaps a README.md in this folder to explain what `simpleprophet` is.

File: simpleprophet/data.py
Comment (jklukas) [inline simpleprophet/data.py]: There's a whole lot of lowerCamelCase going on here. I would love to see this all match python conventions of snake_case for function/method/variable names and UpperCamelCase only for classes.

But consistency within the module is most important, so I respect a decision to hold off on that.

File: simpleprophet/__init__.py
Comment (jklukas) [reply simpleprophet/__init__.py:1]: It would be helpful for reviewing to have a sense of how this library is invoked. What's the function/functions that you call to make a prediction and persist it?

**Review by saptarshiguha — APPROVED** [review]
From a DS pov, I'd add many many comments. 
a) a little bit of thought dump of how you arrived at model parameters
b) what is going on in validation, the type of validity checks
c) the dates and any mysterious dates.

File: simpleprophet/data.py
Comment (jmccrosky) [reply simpleprophet/data.py]: Fixed!

File: simpleprophet/data.py
Comment (jmccrosky) [reply simpleprophet/data.py]: Fixed!

**Comment by jmccrosky** [comment]
I've improved the naming convention and added a README.  I agree that more documentation and comments are needed, but don't want to block the productionizing work, so unless either of you have major concerns, will push this.

File: simpleprophet/data.py
Comment (jklukas) [inline simpleprophet/data.py:5]: Example module-level docstring would be here, right above imports.

```
"""
Tools for getting metric actual or forecast data from BigQuery.
"""
```

File: simpleprophet/README.md
Comment (jklukas) [inline simpleprophet/README.md:7]: These would be great as module-level docstrings either instead of or in addition to the docs here. Example below.

**Review by jklukas — APPROVED** [review]
Let's merge this and I can start taking a look at dockerizing.

File: simpleprophet/README.md
Comment (jmccrosky) [reply simpleprophet/README.md:7]: Merging to unblock you, but will make those changes.

**Pull request merged**
